### DISCUSSION TITLE
Build using cmake or meson.

### DISCUSSION THREAD

**Discussion by moilong** [hyprwm/Hyprland#1465] [General]
Im using hyprland in voidlinux for a while. When I build using cmake from main branch it use hyprland.conf. But when I build using meson, hyprlandd.conf needed. Is this normal.

**Comment by vaxerski** [comment]
@fufexan did you fuck up the meson build to use debug by default

**Reply by fufexan** [reply]
No. Looks like meson builds in [debug by default](https://mesonbuild.com/Builtin-options.html#core-options).

@moilong try this command instead `meson setup --wipe build -Dbuildtype=release -Ddebug=false`.

**Reply by moilong** [reply]
Will do tommorrow and report. Thanks much.

**Reply by vaxerski** [reply]
@fufexan can you make it release by default in the meson.build?

Nothing against even forcing release, the debug builds should be done with cmake regardless.

**Reply by moilong** [reply]
@fufexan that command returns `Directory does not contain a valid build tree:
/home/void/Hyprland/build`
Update: Forget that.  I got it wrong. Your command builds correct. First do a regular `meson build` then issue your command on the same directory. Thanks.

**Reply by fufexan** [reply]
@moilong replace `build` with `_build` and try again.
